bagelbids/tests/test_cli_pheno.py

```diff
@@ -155,3 +155,30 @@ def test_that_output_file_contains_name(runner, test_data, tmp_path):
         pheno = json.load(f)
 
     assert pheno.get("label") == "my_dataset_name"
+
+
+def test_diagnosis_and_control_status_handled(runner, test_data, tmp_path):
+    runner.invoke(
+        bagel,
+        [
+            "--pheno",
+            test_data / "example6.tsv",
+            "--dictionary",
+            test_data / "example6.json",
+            "--output",
+            tmp_path,
+            "--name",
+            "my_dataset_name",
+        ],
+    )
+
+    with open(tmp_path / "pheno.jsonld", "r") as f:
+        pheno = json.load(f)
+
+    assert (
+        pheno["hasSamples"][0]["diagnosis"][0]["identifier"]
```

Comment:
is the `["diagnosis"][0]` here because participants can (eventually) have >1 diagnosis?

Reply:
Not really, it's more that diagnosis will have a list of values so I need to index it to assert over a value.

That said, it is also true that we currently only handle one value per attribute so we can expect that the list has only one element for now.